src/backends/torch/torchlib.cc

```diff
@@ -898,21 +968,23 @@ namespace dd
 
     bool lstm_continuation = false;
     TInputConnectorStrategy inputc(this->_inputc);
+    if (_template.find("nbeats") != std::string::npos)
```

Comment:
I'm thinking having a `nbeats_native_template_inputconn_config()` function somewhere. Possibly into the template actually, i.e. outside of `torchlib`.

Reply:
I am puttinh this as virtual into module._native/ native_net / nbeats , native_net class is there exactlty for doing so